examples/community/README.md

```diff
@@ -371,3 +371,55 @@ for i in range(4):
 for i, img in enumerate(images):
     img.save(f"./composable_diffusion/image_{i}.png")
 ```
+
+### Imagic Stable Diffusion
+Allows you to edit an image using stable diffusion. 
+
+
+```python
+import torch as th
+import numpy as np
+import requests
+from PIL import Image
+from io import BytesIO
+import torch
+from diffusers import DiffusionPipeline, DDIMScheduler
+
+has_cuda = th.cuda.is_available()
+device = th.device('cpu' if not has_cuda else 'cuda')
+
+pipe = DiffusionPipeline.from_pretrained(
+    "CompVis/stable-diffusion-v1-4",
```

Comment:
with safety checker it's returning black images, let's pass None here so the example will work.

```suggestion
    "CompVis/stable-diffusion-v1-4",
    safety_checker=None,
```